docs/tutorials/educators/intro.ipynb

```diff
@@ -99,13 +99,24 @@
    "metadata": {
```

Comment:
<p>let's add a comment here that this doesn't generalize to situations where there are empty <code>Moment</code>s  between the operations. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3967/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='165'/>

Reply:
added a note.